packages/library/package.json

```diff
@@ -8,7 +8,7 @@
   ],
   "scripts": {
     "test": "vitest",
-    "e2e": "playwright install --with-deps chromium && playwright test"
+    "e2e": "playwright install --with-deps chromium && playwright test --workers=1"
```

Comment:
the workers option can be removed. I have set this with the playwright config file.